corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -259,15 +259,23 @@ hqDefine('registration/js/new_user.ko', [
         };
 
         self.isStepOneValid = ko.computed(function () {
+            var isPasswordValid = (
+                self.password() !== undefined
+                && self.password.isValid()
+                && self.passwordDelayed.isValid()
```

Comment:
Is it necessary to evaluate these when `self.isSso()` is true? Would it make sense to put this code in an `else` clause of `if (self.isSso())` below?

Reply:
i just thought this way was a lot more readable since there are a lot of other chained booleans below.

Reply:
ah i think i misunderstood. sure